app/authoring/index.md

```diff
@@ -103,32 +102,32 @@ Yeoman offers a base generator which you can extend to implement your own behavi
 In the generator's index.js file, here's how you extend the base generator:
 
 ```js
-var generators = require('yeoman-generator');
+var Generator = require('yeoman-generator');
 
-module.exports = generators.Base.extend();
+module.exports = class extends Generator {};
 ```
 
-The `extend` method will extend the base class and allow you to provide a new prototype. This functionality comes from the [Class-extend](https://github.com/SBoudrias/class-extend) module and should be familiar if you've ever worked with Backbone.
-
 We assign the extended generator to `module.exports` to make it available to the ecosystem. This is how we [export modules in Node.js](https://nodejs.org/api/modules.html#modules_module_exports).
 
+If you need to support ES5 environment, the static `Generator.extend()` method can be used to extend the base class and allow you to provide a new prototype. This functionality comes from the [Class-extend](https://github.com/SBoudrias/class-extend) module and should be familiar if you've ever worked with Backbone.
```

Comment:
@zckrs Here is the info how to extend the generator with ES5 syntax. But keep in mind that with end of life of Node `0.12` end of this month there is literally no need not to use ES6 features which came with node 4.

See: http://node.green/ for details 😘